javascript/javascript2/week3/Exercises/main.js

```diff
@@ -0,0 +1,75 @@
+// variables 
+// counter 
+const body = document.querySelector('body');
+const counterBtn = document.querySelector('button')
+let count = -1
+// delayBtn
+const delayBtn = document.getElementById('delay')
+let message;
+const delayMessage = document.getElementById('delayMessage');
+//mouseposition
+let coords;
+let mousePositionList = [];
+//const averageCalculater= (preValue, currValue) => ((preValue + currValue) / mousePositionList.length)
+
+
+
+
+counterBtn.addEventListener('click', () => {
+    count += 1
+    counterBtn.innerHTML = `counter : ${count}`;
+})
+
+function logMessage() {
+    message = 'log in 3 seconds';
```

Comment:
It would maybe make a bit more sense that the message was a parameter to logMessage